address_book/internal/service/service.go

```diff
@@ -125,3 +137,26 @@ func getFindContactMsg(size int) string {
 	}
 	return "No contacts were found"
 }
+
+func processFindContact(findContacts *[]models.Contact, err error) (*pb.FindContactResponse, error) {
```

Comment:
What is the reason to pass a pointer to slice here?

Reply:
I preferred to pass pointer to the slice instead of copying the slice structure, may be it be redundant